File: TextC/5.c
#include <stdio.h>
int main(){
    int a,b;
    scanf("%d %d",&a,&b);
    int MA,MB;
    MA=a;
    MB=b;
    int temp;
    while(b>0){
        temp=a%b;
        a=b;
        b=temp;
    }
    printf("最小公倍数:%d",MA/a);
    return 0;
}

File: TextC/5.py
a,b=eval(input())
MA,MB=a,b
while b>0:
    temp=a%b
    a=b
    b=temp

if MA/a>MB/a:
    print("最小公倍数数{}".format(MA/a))
else:
    print("最小公倍数数{}".format(MB/a))